src/api/archive_unstable_storageDiff.md

```diff
@@ -0,0 +1,114 @@
+# archive_unstable_storageDiff
+
+**Parameters**:
+
+- `currentHash`: String containing a hexadecimal-encoded hash of the header of the block whose storage difference will be retrieved. The storage difference is calculated between the `currentHash` block and the parent of the `currentHash` block.
+
+- `fromHash` (optional): String containing a hexadecimal-encoded hash of the header of the block from which the storage difference will be calculated. When this parameter is provided, the storage difference is calculated between the `currentHash` block and the `fromHash` block. If this parameter is not provided, the storage difference is calculated between the `currentHash` block and the parent of the `currentHash` block.
+
+- `items` (optional): JSON object containing the following fields:
+
+    ```json
+    "items": {
+        "prefixes": [
+            {
+                "key": "0x...",
+                "type": "value" | "hash" | "none",
+            },
+        ],
+        "excludeKeyPrefixes": [
+            "0x...",
+        ],
+        "childTrie": "0x...",
+    }
+    ```
+
+  If the `items` parameter is not provided, the storage difference is calculated for all keys of the main storage trie.
+  - `prefixes` (optional): Array of JSON objects describing how the storage difference will be calculated. Each object contains the following fields:
+    - `key`: String containing the hexadecimal-encoded key prefix under which the storage difference is calculated.
+    - `type`: String equal to one of: `value`, `hash`, `none`.
+  - containing the key prefixes for which the storage difference will be calculated. If this parameter is not provided, the storage difference is calculated for all keys.
+  - `excludeKeyPrefixes` (optional): Array of strings containing the key prefixes for which the storage difference will not be calculated. If this parameter is not provided, the storage difference is calculated for all keys.
```

Comment:
You don't "calculate" a storage diff. You probably mean that you exclude some keys from the result sent to the JSON-RPC client.

Reply:
Have remove the `excludeKeyPrefixes` to simplify the API a bit and have adjusted the `key` field to specify that `Only the storage entries whose key starts with the provided prefix are returned.`, thanks!